app/components/public/session-item.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  classNames: ['ui', 'accordion', 'segment']
```

Comment:
The `accordion` class can be removed here. This will fix the extra space on top.

<img width="1024" alt="screen shot 2017-05-23 at 11 59 49 pm" src="https://cloud.githubusercontent.com/assets/2404372/26369982/fca964a4-4013-11e7-845e-7c4e477bc53c.png">